### DISCUSSION TITLE
Reference for BioSimulations paper?

### DISCUSSION THREAD

**Discussion by jreadey** [biosimulations/biosimulations#4584] [General]
Hey, is there an updated version of this paper: https://www.hdfgroup.org/2022/02/biosimulations-a-platform-for-sharing-and-reusing-biological-simulations/?  What can people use for a citiation?

**Comment by jonrkarr** [comment]
We've published papers on [BioSimulators](https://academic.oup.com/nar/article/50/W1/W108/6582178) and [runBioSimulations](https://academic.oup.com/nar/article/49/W1/W597/6279838). A paper on BioSimulations is still in progress.
